crates/nu-command/tests/commands/headers.rs

```diff
@@ -30,6 +30,19 @@ fn headers_adds_missing_column_name() {
     assert_eq!(actual.out, r#"["r1c1","r2c1"]"#)
 }
 
+#[test]
+fn headers_handles_missing_values() {
```

Comment:
could we use an easier to read test here, imo hard to understand all these x, y and zs :confused: 

what about
```rust
#[test]
fn headers_handles_missing_values() {
    let actual = nu!(
    cwd: "tests/fixtures/formats", pipeline(
        "
            [{x: a, y: b}, {x: 1, y: 2}, {x: 1, z: 3}]
            | headers
            | to nuon --raw
        "
    ));

    assert_eq!(actual.out, "[{a: 1, b: 2}, {a: 1}]")
}
```
which is the same in the end? :yum:

Reply:
see the thread below with the change suggestion :wink: